Update hand-menu.md (#1)
cre8ivepark/mixed-reality: mixed-reality-docs/hand-menu.md

<br>

## Hand menu with large and complex UI controls
<img src="images/UX/HandMenu_SizeExample.png" alt="HoloLens perspective of a menu system that always faces the user" width="940px">

> It is recommended to limit the number of buttons or UI controls on hand-attached menus. This is because extended interaction with a large number of UI elements can cause arm fatigue. Instead of keeping the menu leashed to the hand, provide an easy way for the user to world lock the menu. One technique we recommend are to world-lock when the hand drops or flips away from the user. A second technique is to allow the user to grab the menu with the other hand. When the user releases the menu, the menu should world lock. This way, you can interact with various UI elements comfortably and confidently over an extended period of time. 

Author: ```suggestion
It is recommended to limit the number of buttons or UI controls on hand-attached menus. This is because extended interaction with a large number of UI elements can cause arm fatigue. Instead of keeping the menu leashed to the hand, provide an easy way for the user to world lock the menu. One technique we recommend is to world-lock then menu when the hand drops or flips away from the user. A second technique is to allow the user to grab the menu with the other hand. When the user releases the menu, the menu should world lock. This way, a user can interact with various UI elements comfortably and confidently over an extended period of time. 
```
